html-css/week1/styles.css

```diff
@@ -0,0 +1,1240 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
```

Comment:
This files contains a few, albeit minor, issues, and I would recommend you running it through a CSS validator (perhaps [this one](https://jigsaw.w3.org/css-validator/) which finds 15 errors and 6 warnings). 🔍

Reply:
Great tool.
find and removed all the issues. 
thanks  a lot.